CONTRIBUTING.md

```diff
@@ -63,12 +128,98 @@ $ npm test -- --config jest.config.json
 # etc
 ```
 
-### Run coverage
+#### Run coverage
+
+As the Boy Scout Rule states, you are expected to leave the code better than you found it. To do so, please check that your code is properly covered by meaningful tests.
 
 ```sh
 npm test -- --coverage
 ```
 
+#### Browser testing
+
+You can find the list of supported browser in the [List of supported browsers](./README.md#list-of-supported-browsers) section of the README.
+
+If you are developping on a Mac, you will find some Virtual Machines to help you test, on [Microsoft Edge Tools](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/) webiste.
+
+### Documentation
+
+After working on a component, you are expected to update the documentation.
+
+Documentation can be found in the [Design System](https://github.com/AxaGuilDEv/design-system) repository.
+
+Do not forget to update the component status page, located in this repository at:
+
+```text
+└─ src/
```

Comment:
Est ce que c'est vraiment l'endroit pour préciser cela ?

Reply:
I'd root for that, even more since the code was split in another repo :) Not everybody have the history.

Reply:
Le code n'est pas splitté en plusieurs repo. On a d'un côté un site de présentation des guidelines et de l'autre (ici) l'implémentation des ces guidelines en React. 2 choses totalement différente

Reply:
Those 2 things were in the same repo before, that's why I call it a split :-D

Anyway, the end result: if you create / update a component here, you have to update the guideline as well, and it's a job to do in another repo. Therefore, if you are a newbie, like me, you are happy to know it.

I think you are right, I should not put the path to this page (it may change one day). Is it ok to keep only a reminder?